src/pages/Users/SignIn.js

```diff
@@ -1,8 +1,37 @@
 import React, { Component } from 'react';
+import Header from '../../components/Header/Header';
+import './signin.scss';
 
 class SignIn extends Component {
   render() {
-    return <div>null</div>;
+    return (
+      <>
+        <Header />
+        <article className="signIn">
```

Comment:
최상위 태그의 클래스네임은 컴포넌트이름과 동일하게 해주시는게 다른 컴포넌트와 클래스명이 겹칠 확률이 적습니다
```suggestion
        <article className="SignIn">
```